docs/content/en/docs/install/upgrade.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Upgrade
+description: How to upgrade to the latest version of the Lifecycle Toolkit
+icon: concepts
+layout: quickstart
+weight: 45
+hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+To upgrade from the previous version of the Keptn Lifecycle Toolkit to the latest,
+run the same command sequence used to install KLT:
+
+```shell
+helm repo add klt https://charts.lifecycle.keptn.sh
+helm repo update
+helm upgrade --install keptn klt/klt \
```

Comment:
did we test this? AFAIK upgrade may fail due to crds...

Reply:
IMO this may be confusing: if I had a version installed via manifest and run this, helm will fail

Reply:
This iteration just copies the info from the existing docs into the new structure.  It definitely needs to be revisited -- other comments question whether we should even document how to install from a manifest or whether we should mention the cert-manager installation.  And we definitely need to tell people how to upgrade using the helm chart if they previously installed with the manifest.  Also probably need to discuss the cert-manager -- do you have to use our cert manager, can our cert-manager co-exist on the same cluster with another cert-manager that may be instaleld for Argo or Flux...  Lots of stuff that needs to be clarified and expanded.  This PR is just the first draft of Phase 1 -- we still have muc work to do but baby steps...